### PR TITLE
orca init

### PR DESCRIPTION
**Pull request by cyita**

### CODE REVIEW

File: pyzoo/test/zoo/xshard/test_ray_pandas.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 import pytest
```
Comment (jason-dai) [inline pyzoo/test/zoo/xshard/test_ray_pandas.py:19]: move the file for test/zoo/orca/data/

File: pyzoo/zoo/examples/xshard/ray_pandas.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 from bigdl.util.common import get_node_and_core_number
```
Comment (jason-dai) [inline pyzoo/zoo/examples/xshard/ray_pandas.py:21]: move it to zoo/examples/orca/data/

**Comment by jason-dai** [comment]
@jenniew @cyita which one to merge first https://github.com/intel-analytics/analytics-zoo/pull/2304 or https://github.com/intel-analytics/analytics-zoo/pull/2305?

**Comment by jenniew** [comment]
> @jenniew @cyita which one to merge first #2304 or #2305?

I think #2304 may be merged first if it is ready.
I'm still running jenkins.

**Comment by cyita** [comment]
jenkins passed http://10.239.47.210:18888/view/ZOO-PR-PIPELINE/job/ZOO-PR-Validation/3315/

File: pyzoo/test/zoo/xshard/test_ray_pandas.py (hunk 1 above)
Comment (cyita) [reply pyzoo/test/zoo/xshard/test_ray_pandas.py:19]: Fixed

File: pyzoo/zoo/examples/xshard/ray_pandas.py (hunk 2 above)
Comment (cyita) [reply pyzoo/zoo/examples/xshard/ray_pandas.py:21]: Done

**Comment by jason-dai** [comment]
Change `DataShards.apply` to `DataShards.transform_shard` in a follow-up PR

**Pull request merged**